### PR TITLE
update the max job dimensions

### PR DESCRIPTION
**Pull request by corindwyer**
### Description of the Change
* Update the max job dimensions.

### CODE REVIEW

File: titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobConfiguration.java [hunk 1]
```diff
@@ -80,7 +80,7 @@
      * An upper bound on disk (megabytes) a single container may allocate. The actual limit may be lower, as it also depends
      * on instance types available in a tier.
      */
-    @DefaultValue("999000")
+    @DefaultValue("1550000")
```
Comment (rostyboost) [inline titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobConfiguration.java:83]: (just a reminder that the Docker image size is counted in this budget, which is why we currently set the min at 10GB)

**Review by tbak — APPROVED** [review]

File: titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobConfiguration.java [hunk 2]
```diff
@@ -70,7 +70,7 @@
      * An upper bound on memory (megabytes) a single container may allocate. The actual limit may be lower, as it also depends
      * on instance types available in a tier.
      */
-    @DefaultValue("472000")
+    @DefaultValue("760000")
```
Comment (rostyboost) [inline titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobConfiguration.java:73]: should we leave some slack for executor operations + spike during coredumps?

Comment (corindwyer) [reply titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobConfiguration.java:73]: I left a couple of G but I could reduce it to be 750000 so there is > 10GB buffer.

File: titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobConfiguration.java (hunk 1 above)
Comment (corindwyer) [reply titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobConfiguration.java:83]: yes

File: titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobConfiguration.java (hunk 2 above)
Comment (rostyboost) [reply titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobConfiguration.java:73]: Yes, `750000` might be better

**Review by rostyboost — APPROVED** [review]

**Review by andrew-leung — APPROVED** [review]
It may be worth some comments as bread crumbs for why the default values are what they are (e.g., "based on m5.metal advertised values") for the next person.

**Pull request merged**